packages/upstream/conf-time.1/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "1.2"
+maintainer: "Gabriel Scherer <gabriel.scherer@gmail.com>"
+authors: ["POSIX"]
+homepage: "https://github.com/ocaml/opam-repository/"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "https://github.com/ocaml/opam-repository.git"
+license: "GPL"
+build: [["which" "time"]]
```

Comment:
Is this package used by anything? Seems odd to have GPL license for something so simple.

Reply:
You are right, we can actually get rid of it. AFAIR, it was used by a package along with conf-which but it was meanwhile updated or removed